workflow/tests/base_results/results_ashrae_140.csv

```diff
@@ -1,27 +1,27 @@
 Test Case,Annual Heating Load [MMBtu],Annual Cooling Load [MMBtu]
-L100AC.xml,51.41,N/A
```

Comment:
Addressed in https://github.com/NREL/OpenStudio-HPXML/pull/1340/commits/d7cb726c5e6aad041f55bb24d829e4a8be744442. I ended up going with option 2 above, even though option 1 was my preference because it seemed to fit the description more correctly. Bringing the house back down to ground level yielded the same results, so I went with that, left the `NoWind` thing in, but commented out.

Reply:
I looked at this briefly. The reason that your commented line has no effect is because it isn't changing the IDF. The code is already setting NoWind a few lines further below in the code ([here](https://github.com/NREL/OpenStudio-HPXML/blob/mobile_home_belly_ducts/HPXMLtoOpenStudio/measure.rb#L662)). So you can safely remove the commented line.

Presumably all we are seeing then is a difference in wind speed due to the height of the conditioned zone (and not a difference in the raised floor heat transfer due to wind). I see nothing in ASHRAE 140 that specifies the above-grade height of the raised floor, so I suggest we stick with what we've been using (zero).